code/game/objects/structures/window.dm

```diff
@@ -524,6 +524,13 @@
 	basestate = "fwindow"
 	maxhealth = 30
 
+/obj/structure/window/reinforced/podded
+	name = "podded window"
+	desc = "It looks rather strong over. Looks like it might take a few less hits then a normal reinforced window."
```

Comment:
```suggestion
	desc = "It looks rather strong over. Looks like it might take a few less hits than a normal reinforced window."
```

а что в первом предложении вообще написано я так и не понял, нужен эксперт

Reply:
Пофикшено